README.md

```diff
@@ -22,33 +23,96 @@ The supportet visualizations are:
+You are here | ![[UnfoldMakie.jl](unfoldmakie.com)](https://github.com/unfoldtoolbox/Unfold.jl/assets/10183650/37af35a0-c99c-4374-827b-40fc37de7c2b)
+Julia interface to MNE-Python via PythonCall | [PyMNE.jl](https://github.com/beacon-biosignals/PyMNE.jl/tree/main)
+</details>
+
+## Install
+
+### Installing Julia
+
+<details>
+<summary>Click to expand</summary>
+
+The recommended way to install julia is [juliaup](https://github.com/JuliaLang/juliaup).
+It allows you to, e.g., easily update Julia at a later point, but also test out alpha/beta versions etc.
+
+TL:DR; If you dont want to read the explicit instructions, just copy the following command
+
+#### Windows
+
+AppStore -> JuliaUp,  or `winget install julia -s msstore` in CMD
+
+#### Mac & Linux
 
-The code files can be found in the `src` folder.
+`curl -fsSL https://install.julialang.org | sh` in any shell
+</details>
 
-### Documentation Notes
+### Installing Unfold
 
-The documentation can be found in the `doc` folder. We use `Documenter.jl` and plan to use `Literate.jl` for future usage.
-In the `src` folder contained within it exists as "raw" `.md` files and in the `build` folder contained within it exists as `.html` after running the `make.jl` file.
-These are specifically the "index" file with more files in sub folders.
+```julia
+using Pkg
+Pkg.add("UnfoldMakie")
+```
+
+## Quickstart
+
+```julia
+using UnfoldMakie
+using CairoMakie
+
+# TBA
+```
+
+## Contributions
+
+Contributions are very welcome. These could be typos, bugreports, feature-requests, speed-optimization, new solvers, better code, better documentation.
+
+### How-to Contribute
+
+You are very welcome to raise issues and start pull requests!
+
+### Adding Documentation
+
+1. We recommend to write a Literate.jl document and place it in `docs/_literate/FOLDER/FILENAME.jl` with `FOLDER` being `HowTo`, `Explanation`, `Tutorial` or `Reference` ([recommended reading on the 4 categories](https://documentation.divio.com/)).
+2. Literate.jl converts the `.jl` file to a `.md` automatically and places it in `doc/src/_literate/FILENAME.jl`.
+3. Edit [make.jl](https://github.com/unfoldtoolbox/Unfold.jl/blob/main/docs/make.jl) with a reference to `doc/src/_literate/FILENAME.jl`
 
 ## Citation
-If you make use of theses visualizations, please cite [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6531996.svg)](https://doi.org/10.5281/zenodo.6531996)
 
-## Authors
+If you make use of theses visualizations, please cite:
 
-It was authored by `Benedikt Ehinger` and has been worked upon by students and doctoral researchers within the context of a university project (`Fachpraktikum:  Methods in Computational EEG`) at the University of Stuttgart.
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.6531996.svg)](https://doi.org/10.5281/zenodo.6531996)
 
-The student names are:
-- Baumgartner, Daniel
-- Döring, Sören
-- Gärtner, Niklas
+## Contributors (alphabetically)
 
```

Comment:
you are a contributor too :)

Reply:
So far only because I updated the readme, no? Didn't think that would justify being on the list 👀